certificate/forms.py

```diff
@@ -16,3 +17,23 @@ class FeedBackForm(forms.Form):
     #            (max_length=30, required=False)
     #state = forms.CharField\
     #            (max_length=128)
+
+email_subject_choice = [
+('Certificate not Awarded','Certificate not Awarded'),
+('Change in Name','Change in Name'),
+('Issue with Workshop Date','Issue with Workshop Date'),
+('Invalid Email Address','Invalid Email Address'),
+('Others','Others')]
+ws_type_choice = [
+('iscp','Introduction to Scientific Computing using Python(ISCP)'),
+('3day','Basic Programming using Python'),
+('sel','Self Learning(Basics of Python)')]
```

Comment:
Should we have other here?

Reply:
Since the workshop categories are fixed , did not give Others option.